packages/smooth_app/lib/pages/prices/product_price_add_page.dart

```diff
@@ -65,6 +66,7 @@ class ProductPriceAddPage extends StatefulWidget {
             locations: osmLocations,
             initialProduct: product,
             currency: currency,
+            multipleProducts: multiProduct,
```

Comment:
```suggestion
            multipleProducts: multipleProducts,
```

Please try to keep consistency between classes.